### DISCUSSION TITLE
Slidev or Unocss port or vanilla css for website

### DISCUSSION THREAD

**Discussion by yodatak** [catppuccin/catppuccin#2421] [Port Requests]
### Will you make the port?

- [ ] Yes, I'm willing to make the port

### How can the suggested port be themed?

https://sli.dev/guide/syntax.html#embedded-styles
https://github.com/Cap-go/tailwindtouno

### Submission Guidelines

- [X] I have read and followed Catppuccin's submission guidelines.

**Comment by uncenter** [comment]
Duplicate of both https://github.com/catppuccin/catppuccin/discussions/2155 and https://github.com/catppuccin/catppuccin/discussions/1943.
